omod/src/main/java/org/openmrs/template/web/controller/FooterFullController.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.template.web.controller;
+
+
+import org.openmrs.api.context.Context;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+
+public class FooterFullController {
+
+
+	@RequestMapping("/template/footerFull.htm")
+	public String footerFull(Model model) {
+
+		String footerExtraData = Context.getRuntimeProperties()
+				  .getProperty("legacyui.footer.extradata", "");
+			if (footerExtraData.equals("legacyui.footer.extradata")) {
+				footerExtraData = null;
+			}
```

Comment:
What's this?

Reply:
let me first get the screen shorts from the instance and get back here